commands/addons/attach.js

```diff
@@ -1,44 +1,44 @@
-'use strict';
+'use strict'
 
-let cli      = require('heroku-cli-util');
-let co       = require('co');
+let cli = require('heroku-cli-util')
+let co = require('co')
 
-function* run (context, heroku) {
-  let app = context.app;
-  let addon = yield heroku.get(`/addons/${context.args.addon_name}`);
+function * run (context, heroku) {
+  let app = context.app
+  let addon = yield heroku.get(`/addons/${context.args.addon_name}`)
 
-  function createAttachment(app, as, confirm) {
+  function createAttachment (app, as, confirm) {
```

Comment:
wow... really? That seems surprisingly unconventional in basically any C-style lang